app/helpers/ncr/work_orders_helper.rb

```diff
@@ -1,9 +1,8 @@
 module Ncr
   module WorkOrdersHelper
     def approver_options
-      # @todo should this list be limited by client/something else?
       # @todo is there a better order? maybe by current_user's use?
-      User.order(:email_address).pluck(:email_address)
+      User.where(client_slug: 'ncr').order(:email_address).pluck(:email_address)
```

Comment:
there is a unit test on the helper class itself in `spec/helpers/ncr/work_orders_helper_spec.rb` which has been changed to reflect this new feature.

I got the impression that we were trying to keep the model class limited to business-rule-specific code, and put presentation in decorators or helpers. This method feels like the latter and has test coverage.

That seem ok to you?